src/core/environment.js

```diff
@@ -369,6 +369,35 @@ p5.prototype.noCursor = function() {
   this._curElement.elt.style.cursor = 'none';
 };
 
+/**
+ * If the was created in WebGL mode, then `weglVersion` will indicate which
+ * version of WebGL it is using. It will try to create a WebGL2 canvas unless
+ * you have set `p5.disableWebGL2 = true`, and will fall back to WebGL1 if
+ * WebGL2 is not available.
+ *
+ * `webglVersion` will always be either `WEBGL2` or `WEBGL`.
+ * @property {String} webglVersion
```

Comment:
I think many of the function ones are functions because they use the pattern where `frameRate(value)` is a setter while `frameRate()` is a getter. It makes sense from an OOP standpoint for `getTargetFrameRate()` to be a function so that you can't accidentally set the value, but it's also a bit of a deviation from seemingly the rest of p5, where read-only values are just properties (`mouseX`, `pmouseX`, `frameCount`, etc.)

I think if we make the webgl version settable via `setAttributes`, having its value exposed as a property would probably be most consistent then? But I don't think any other attributes are later readable (via a documented API anyway), so whatever we choose to do here can maybe set the precedent.

> What will this value return if the p5 instance is not in webGL mode?

Right now it's `WEBGL2`, which is probably confusing. Do you think it should be something like `undefined` or `P2D`?

Reply:
I think returning the current renderer at `P2D` makes sense to me. 

Lets go ahead with returning the exposed value as a property then. I suppose most of the other ones aren't available simply because it probably isn't very useful information to have. I guess if we wanted to we could implement a `getAttributes()` function that would return either a specific key or the attributes object of the current renderer. 

Probably better to keep things simple for now and we can revisit those other attributes at a later time if the issue arises.